participants/matin-cloud-pan.md

```diff
@@ -0,0 +1,63 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*Project name is matin-cloud-pan , and our team is Matin Cloud (麦田云际) . Now , we are working on filecoin and IPFS*
+
+## (1) Brief Project Description
+
+*This project mainly stores various data, such as pictures, audio and video, etc.*
+
+## (2) Link to Project UI
+
+*Web: http://pan.mtfs.top*
+
+*Android & iOS: https://app.mtfs.top/down/pan-down/downpage.html*
+
+## (3) What does your application/UI do?
+
+*Provide individuals with secure data storage services on filecoin, and the live broadcast function will be developed in the future.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+* *
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*We will collect data such as videos, audios, pictures, etc. We have the power to process the data, and the data does not contain any sensitive or illegal information.*
```

Comment:
What is your mechanism to ensure that the data is not senstiive or illegal?

Reply:
We use our data review services to verify the legality of data.

Reply:
And the user who uploads the data has the ownership of the data. We will conduct a certain review (machine or manual) on the uploaded video data to ensure the legality and validity of the data.